dev-infra/ngbot/verify.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {readFileSync} from 'fs';
+import {resolve} from 'path';
+import {parse as parseYaml} from 'yaml';
+
+import {getRepoBaseDir} from '../utils/config';
+import {error, info} from '../utils/console';
+
+export function verify() {
+  /** Full path to NgBot config file */
+  const NGBOT_CONFIG_YAML_PATH = resolve(getRepoBaseDir(), '.github/angular-robot.yml');
+
+  /** The pull approve config file. */
+  const ngBotYaml = readFileSync(NGBOT_CONFIG_YAML_PATH, 'utf8');
+
+  try {
+    // Try parsing the config file to verify that the syntax is correct.
+    parseYaml(ngBotYaml);
+    info('NgBot YAML config is valid');
+  } catch (e) {
+    error(e);
```

Comment:
Can we have a line expressing directly that the config was not validated?

```suggestion
    error(`${red('!')} Invalid NgBot YAML config`);
    error(e);
```